### DISCUSSION TITLE
what's the best file structure for the Besttime

### DISCUSSION THREAD

**Discussion by mohadmn03** [mohadmn03/Besttime#2] [Q&A] [answered]
Please share the best file structure that you used for such a project

**Comment by JacobAthanasiou** [comment, marked as answer]
Hi Moha!

I've been there, this should be very helpful
https://reactjs.org/docs/faq-structure.html

let me know if you needed any more help

**Reply by mohadmn03** [reply]
Wow Thank you so much
